docs/planning/frontend/src/pages/ReviewerPage.jsx

```
@@ -0,0 +1,250 @@
+          ) : (
+            <p className="text-gray-500">No recommendations available yet.</p>
+          )}
+
+          {/* AI Suggestions */}
+          {llmStatus?.configured && (
+            <div className="mt-6 pt-6 border-t">
+              <button
+                onClick={() => setShowSuggestions(!showSuggestions)}
+                className="text-perfume-600 hover:text-perfume-700 font-medium"
+              >
+                🤖 Get AI Suggestions for New Fragrances to Try
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)